### ISSUE TITLE
command line parameters: marxan now only accepts 'input.dat'?

### ISSUE THREAD

**Issue by remi-daigle** [Marxan-source-code/marxan#8]
Is it no longer possible to give marxan arguments (i.e. the input filepath) when calling from the command line? Older versions would allow you to specify the the path of the input file. For example, with marxan 2.43, from a normal marxan database, you could do:

```
Marxan_64.exe
```

and the software would look for the default `input.dat`  and if you wanted to use an alternatively named file, you could use alternate filenames or filepaths:

```
Marxan_64.exe alternativenameinput.dat
```
or
```
Marxan_64.exe C:/users/Remi/alternativepath/input.dat
```

Unfortunately, with the new version (4.0.5), when I run either of the above options, it gives the following error:

```
        Marxan v 4.0.5

   Spatial Prioritization via Annealing

   Coded by Ian Ball, modified by Matthew Watts
   Written by Ian Ball and Hugh Possingham

   Marxan website

https://marxansolutions.org/

Input file input.dat not found
Aborting Program.

Error during Marxan execution.
(base)
```

The new manual suggests you can still give it command line parameters on page 5-29, but I cannot make it work like it used to. Such functionality is really important for people who use marxan from the command line or from R. It also makes it possible for Marxan Connect to call marxan from it's install directory without having to copy the marxan executable in each database folder.

**Comment by adelebai** [comment]
Hi Remi, thank for pointing out this issue! It's a bug and it should be fixed in the latest 4.0.6 build.

**Comment by remi-daigle** [comment]
Perfect, works like a charm with 4.0.6, thanks!
